docs/contribute.rst

```diff
@@ -398,6 +398,12 @@ We require this wrapper as a drop-in replacement of pytest due to quirks of geve
 For running the tests with a more specific usage and invocation, please refer to the `pytest <https://docs.pytest.org/en/stable/usage.html>`__ documentation.
 
 
+We run the test suite in the GitHub CI but only a subset of them since not all are needed and sometimes they suffer of rate limiting. We have some special settings to choose what tests are executed:
```

Comment:
```suggestion
We run the test suite in the GitHub CI but only a subset of them since not all are needed and sometimes they suffer from rate limiting. We have some special settings to choose what tests are executed:
```